main-app/app/scripts/Gameboard/Controller/GameboardController.js

```diff
@@ -0,0 +1,41 @@
+(function () {
+    'use strict';
+    angular.module('Tombola.NoughtAndCrosses.gameboard')
+    .controller('GameboardController',function ($scope, Proxy, playerToggle, GameModel){
+
+        this.gameModel = GameModel;
+
+        this.gameboardTapped = function (gridNumberFromTable) {
+            if (GameModel.gameboard.charAt(gridNumberFromTable) !== '0' || GameModel.currentState === 'Win') {
```

Comment:
Break down into two guard clauses.
The game model would better expose two functions: IsGameInPlay, IsSquareSelected(squareNumber)